Dockerfile

```diff
@@ -3,7 +3,10 @@ FROM php:7.1-fpm-alpine
 RUN apk add --no-cache --virtual .persistent-deps \
 		git \
 		icu-libs \
-		zlib
+		zlib \
+		wget \
+		ca-certificates \
```

Comment:
Agree, if we can keep using alpine, its better. If we can't I can merge as it.

Tell me 👍